src/framework/Framework_Controller_002.test.js

```diff
@@ -0,0 +1,50 @@
+import {
+  getControllerPluginData,
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+
+export default {
+  title: 'Framework Controller test 02',
+  description: 'Tests basic functionality of the controller module',
+  teardown: restartFramework,
+  steps: [
+    {
+      description: 'Get /Service/Controller and validate whether it is Object or not',
```

Comment:
Is no longer Get /Service/Controller, but done through jsonRPC.

Reply:
Removed restartFramework

Reply:
Renamed the description as I really use json RPC in the test case